edu.stanford.protege.json

```diff
@@ -0,0 +1,74 @@
+{
+    "id": "edu.stanford.protege",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk8"],
+    "command": "protege",
+    "finish-args": [
+        "--env=PATH=/app/bin:/app/jre/bin:/usr/bin",
+        "--socket=x11",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": ["/usr/lib/sdk/openjdk8/install.sh"]
+        },
+        {
+            "name": "protege",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/protegeproject/protege-distribution/releases/download/v5.5.0/Protege-5.5.0-platform-independent.zip",
+                    "sha512": "7dac94f2cc9f38c0993ab374abdc70d6dd581f3929b6f5d19e2a19b6df4ffc4154db697964b055451070cf1afc1c6f0bb2ed741751c3a6517f6ea6b050f70a55"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "edu.stanford.protege-16.png"
```

Comment:
that's a lot of icons added. Only 128x128 is required. If there is a SVG it would be nice.

Reply:
I had to extract all those from a `.ico` format that is shipped in the release. I could potentially do that in the build if ImageMagick is part of the SDK.

Reply:
it's not.

Maybe you should submit these so they are part of the ZIP file, with the .desktop and .appdata.xml. After all if they claim to be cross-platform but only ship asset for Windows and macOS there is some sort of omission.

Reply:
but you should only keep the 128x128 here.

Reply:
Even their Linux distribution does not include compatible icons. I'll try to upstream these assets.

Is it okay to let the desktop environment rescale the 128x128 v.s. using what the author intended in the `ico` format? Unfortunately, no SVG is included.

Reply:
128x128 is the minmum size we require on Flathub. It can be either downscaled, taken as is, or from a SVG. There are a few options, in that case keeping only the 128x128 is the option.